spec/fixtures/parameterizing_rules/user_defined.y

```diff
@@ -0,0 +1,42 @@
+/*
+ * This is comment for this file.
+ */
+
+%{
+// Prologue
+static int yylex(YYSTYPE *val, YYLTYPE *loc);
+static int yyerror(YYLTYPE *loc, const char *str);
+
+%}
+%union {
+    int i;
+}
+
```

Comment:
If `number` is a token (terminal), it should be defined with `%token <i> number`, `<i>` is optional in this case.

Reply:
Oh... sorry I missed it 😣 Added in the following commit:
https://github.com/ruby/lrama/pull/285/commits/7ad441cf3b07deafd7fa316b336626d8f6d352e5